style.css

```diff
@@ -779,3 +779,41 @@ main{
     cursor: pointer;
     padding: 0.5rem;
```

Comment:
You can use px here, generally scaling such a small value with rem is not desirable.

Reply:
okay, will keep that in mind!